daml-foundations/daml-ghc/daml-prim-src/GHC/Base.daml

```diff
@@ -57,12 +57,23 @@ infixr 9 .
 const                   : a -> b -> a
 const x _               =  x
 
+-- | A `Functor` is a typeclass for things that can be mapped over (using
+-- its `fmap` function. Examples include `Maybe` and `[]`.
+
+-- For a detailed explanation, see 
+-- http://learnyouahaskell.com/making-our-own-types-and-typeclasses#the-functor-typeclass
```

Comment:
https: links where possible.

Reply:
unfortunately the site does actually use http, not https